lib/web_ui/dev/test_platform.dart

```diff
@@ -689,7 +689,12 @@ class BrowserManager {
     var completer = Completer<BrowserManager>();
 
     browser.onExit.then((_) {
-      throw Exception('${runtime.name} exited before connecting.');
+      // For the cases where we use a delegator such as `adb` (for Android) or
```

Comment:
The closure passed to `.then()` is empty (except the comment). Let's remove it and move the comment outside.

Reply:
done. thanks